package.json

```diff
@@ -38,23 +32,20 @@
     "lint:commit": "ui-scripts --lint-commit",
     "lint:staged": "lint-staged",
     "update:package:list": "lerna run generate:package:list --stream --scope @instructure/instui-config",
-    "commit:package:list": "git add **/package-list.json",
+    "commit:package:list": "git add packages/instui-config/package-lists/**/package-list.json",
     "generate:component": "instui create component",
     "generate:package": "instui create package",
     "commit": "ui-scripts --commit",
-    "install:react": "ui-scripts --install-react",
-    "install:packages": "lerna bootstrap --reject-cycles",
     "bootstrap": "node scripts/bootstrap.js",
     "bootstrap:docs": "npm-run-all clean:modules install:packages clean build",
```

Comment:
this is using `clean:modules` and `install:packages`, but those were removed.

Reply:
Can you please check this too?

Reply:
I'm not sure what you mean, nothing is using those scripts anymore.

Reply:
When I commented this, the `bootstrap:docs` script was net yet removed and it was still using them, but not valid anymore :)